ega-data-api-res/src/main/java/eu/elixir/ega/ebi/reencryptionmvc/domain/Format.java

```diff
@@ -2,6 +2,6 @@
 
 public enum Format {
 
-    PLAIN, AES, AES128, AES256, GPG_SYMMETRIC, GPG_ASYMMETRIC
+    PLAIN, CRYPT4GH
```

Comment:
You should still include AES as target format; even if i is not used at present. GPG can be discarded.